static/css/app.css

```diff
@@ -8089,3 +8089,6 @@ body.infinity-protocol #content-main > div > div.row > div.span3 {
   position: fixed;
   overflow: auto;
 }
+.tobecopied {
```

Comment:
There are direct changes to app.css there.

It'll be overwritten when the less file are compiled. It should be done in less files